Update README with instructions and update default parameters

blacksaturn1 opened the pull request:
1. Update README with instructions
2. Enable tier1 as the default with no command line parameters

blacksaturn1-wpi committed:
1. Update README with instructions
2. Enable tier1 as the default with no command line parameters

arjangupta reviewed (approved):
Cool. Let's just get this project copied over to the common location after this PR and then do any further work in there.

blacksaturn1-wpi committed:
1. Add debug parameter and update README
2. Use debug parameter to display start to goal line in green
3. Use debug parameter to display current position to next action line segment in red
